0.download_data/README.md

```diff
@@ -1,33 +1,11 @@
 # Download CFReT data
 
-The information regarding the CFReT pilot data are as follows:
-
-```
-Assay: Cell Painting
-
-Channels: d0 (Nuclei), d1 (Endoplasmic Reticulum), d2 (Golgi/Plasma Membrane), d3 (Mitochondria), d4 (F-actin)
-
-Cells: Cardiac Fibroblasts
-
-Samples: Cells from three patients that suffered cardiac arrest
-
-Perturbations: One negative control (DMSO) and one drug treatment
-
-Microscope: TBD
-
-Magnification: TBD
-
-Bit-size: 16-bit
-
-Number of images: 990 images
-
-Total memory: 2.42 GB
-```
+In this module, we will present how we download the images for the CFReT project. Currently, the images are stored in a shared DropBox and we include all plates in an `Images` folder within this module.
```

Comment:
The images are not public yet as the collaborators are not ready for that. 

This will likely be fixed in the long term since the plates we have are still pilot data.